en/docs/guides/mfa/mfa-for-management-console.md

```diff
@@ -0,0 +1,97 @@
+# Configuring multi-factor authentication for the management console
+
+## Prerequisites
+You need to [create a service provider](../applications/register-sp.md).
+
+## Configure MFA for the management console
+
+To configure MFA for the management console, initially:
+
+1. On the management console, go to **Main** > **Identity** > **Service Providers** > **List**.
+
+2. Click **Edit** on the service provider you wish to configure.
+
+    Configuration of MFA for the management console contains three configuration sections as follows:
+
+    1. [Inbound authentications configurations](#inbound-authentications-configurations)
```

Comment:
Should these be numbers or bullets? 
Should these be followed in sequence?

Reply:
1 and 2 are okay to be interchanged, but 3 should be done as the last step